lib/session/session.go

```diff
@@ -110,6 +110,48 @@ type Session struct {
 	Moderated bool `json:"moderated"`
 }
 
+// FileTransferParams contain parameters for requesting a file transfer
+type FileTransferParams struct {
+	RequestID string `json:"requestId"`
+	Approved  bool   `json:"approved"`
+	// Response denotes if this is a response to a file transfer request.
+	// If true, the Approved field will be set. True = approved
+	Response bool `json:"response"`
+	// Direction is either upload or download
+	Direction string `json:"direction"`
+	// Location is location of file to download, or where to put an upload
+	Location string `json:"location"`
+	// Filename is the name of the file to be uploaded
+	Filename string `json:"filename"`
+	// Size is the size of the file to be uploaded
+	Size string `json:"size"`
```

Comment:
What would this look like? Why a string?

Reply:
This was vestigial from testing/tinkering. will remove as it's not used